assets/containerisableComponents/payPalContributionButton/payPalContributionButton.jsx

```diff
@@ -6,7 +6,6 @@ import React from 'react';
 
 import Switchable from 'components/switchable/switchable';
 import PaymentError from 'components/switchable/errorComponents/paymentError';
-import SvgPaypalPLogo from 'components/svgs/payPalPLogo';
```

Comment:
Can we delete this SVG if it's no longer used?

Reply:
Yes

Reply:
Deleted here: 8c40f23